steps/operators_test.py

```diff
@@ -45,3 +49,31 @@ def test_concatenate_qubit_operator_lists():
 
     assert group_read == h2_hamiltonian_grouped
 
+def test_get_one_qubit_hydrogen_hamiltonian():
+
+    # Interaction Hamiltonian of H2 (0.74 A, STO-3G) generated using openfermion-psi4
+    constant = 0.7151043387432434
+    # Load one body tensor from .npz file
+    file_handle_1b = np.load('h2_one_body_tensor.npz')
+    one_body_tensor = file_handle_1b.get('arr_0')
+    # Load one body tensor from .npz file
+    file_handle_2b = np.load('h2_two_body_tensor.npz')
+    two_body_tensor = file_handle_2b.get('arr_0')
+    # Assemble interaction operator
+    h2_hamiltonian_int = InteractionOperator(constant=constant, one_body_tensor=one_body_tensor, two_body_tensor=two_body_tensor)
```

Comment:
I guess you could also save the InteractionOperator from openfermion-psi4 directly as a json and use load_interaction_operator, but maybe it doesn't matter that much.

Reply:
I have implemented all the changes in a5dd0fa. I thought CI might be failing because I relied on npz files to store one- and two-body coefficients so I just moved those tensors to the python file (if that's okay).